### ISSUE TITLE
React 15 and Object.assign

### ISSUE THREAD

**Issue by rmdort** [Pomax/react-onclickoutside#79]
React 15 doesnt ship with Object.assign . Getting module not found error

````
Module not found: Error: Cannot resolve module 'react/lib/Object.assign' in ..../node_modules/react-onclickoutside
 @ ./~/react-onclickoutside/decorator.js 2:19-53
````

**Comment by anorudes** [comment]
+1

**Comment by mijamo** [comment]
Related to this: https://github.com/facebook/react/pull/6376

**Comment by jshthornton** [comment]
I've included a fix in this PR: https://github.com/Pomax/react-onclickoutside/pull/80

**Comment by Pomax** [comment]
landed, v4.8.0 pushed to npm
